docs/deploy-dev.md

```diff
@@ -0,0 +1,44 @@
+Augur | dev deployment
+
+___
+## Summary
+
+Unlike production, Augur client is hosted for development. There are two domains available, dev.augur.net and ipfs-dev.augur.net. These sites are hosted from docker container from the dockerfile in augur repository. 
+
+___
+
+## Dockerfile
+
+The dockerfile in augur repository hosts augur client in two ways; port 8001 hosts the build directory via nginx and port 80 is a pass through to the ipfs gateway internal port 8080.
```

Comment:
Why not have nginx be exposed on 80 with 8080 being the direct passthrough to ipfs?